### DISCUSSION TITLE
About compatibility of different versions of forcefield (.pb) files.

### DISCUSSION THREAD

**Discussion by ZhouXY-PKU** [deepmodeling/deepmd-kit#639] [Q&A] [answered]
Hello everyone,

I want to do MD with [Lammps compiled with DeePMD-kit 1.2.2] based on [a force-field file trained with DeePMD-kit 1.1.4] but failed (the log is like 'No attr named 'T' in NodeDef'). Could you please tell me if the two versions of the force-field are incompatible and if the old-version one can be transformed into the new-version one?

Thank you very much!

**Comment by njzjz** [comment, marked as answer]
We haven't provided this feature for r1.1. However, it should be possible in theory.
